pyfenix/__main__.py

```diff
@@ -1,16 +1,28 @@
 """Fenix runner"""
 
-import queue
+import asyncio
+from queue import Queue
 import threading
 
-from tests.doubles import ClientAPIStub, GUIEventHandlerStub
+from tests.doubles import GUIEventHandlerStub
 
 from .client_event_loop import ClientEventLoop
+from .ws_api import WebsocketsAPI
 
-event_queue: queue.Queue = queue.Queue()
-api = ClientAPIStub(event_queue)
-gui = GUIEventHandlerStub(event_queue)
+async def main() -> None:
+    """Run Fenix"""
+    event_queue: Queue = Queue()
```

Comment:
Queue only seems to be used to pass to `WebsocketsAPI`, it might be better to define it only in `WebsocketsAPI` unless there's another planned use for it, or if its used for testing purposes.

Reply:
It is passed to both the GUI and the client event loop as well.